u-root.go

```diff
@@ -90,6 +93,8 @@ func init() {
 	// Flags for the gobusybox, which we hope to move to, since it works with modules.
 	genDir = flag.String("gen-dir", "", "Directory to generate source in")
 
+	// Flag for the new filepath only mode. This will be required to find the u-root commands and make templates work
+	urootSourceDir = flag.String("uroot-source", "", "Path to the locally checked out u-root source tree in case commands from there are desired.")
```

Comment:
Could this default to ./ instead of ""? Then in a lot of your docs above you could delete the -root-source=./ bits.